### PR TITLE
Creates a shorter option for command arguments

### PR DESCRIPTION
**Pull request by Honny1**
This PR creates a shorter option for command arguments.

### CODE REVIEW

File: oval_graph/client.py [hunk 1]
```diff
@@ -201,18 +201,21 @@ def prepare_parser(self):
...
             default=False,
             help="Process all matched rules.")
         self.parser.add_argument(
+            '-p',
             '--hide-passing-tests',
             action="store_true",
             default=False,
             help=(
                 "Do not display passing tests for better orientation in"
                 " graphs that contain a large amount of nodes.(Not implemented)"))
```
Comment (matejak) [inline oval_graph/client.py:216]: If it is not implemented, don't have it here, just comment it out.

**Review by matejak — CHANGES_REQUESTED** [review]
Not all arguments need a long and short option - I recommend to implement this for ones that you expect to be used most. But the proposal is OK.

I have noticed that you have a duplication in arguments parsing code - I recommend to share the code, so consistency of interfaces is maintained automatically.

File: oval_graph/client.py (hunk 1 above)
Comment (Honny1) [reply oval_graph/client.py]: This option has been implemented, but I forgot to remove it.

**Comment by matejak** [comment]
LGTM, merging, I will create issues for the duplication.

**Review by matejak — APPROVED** [review]

**Pull request merged**